students/jemielity_kamil/lesson_09_reading_and_writing_files/regex_search.py

```diff
@@ -0,0 +1,20 @@
+import os
+import re
+
+text_files = []
+
+for file in os.listdir(os.getcwd()):
+    if file.endswith(".txt"):
+        text_files.append(file)
+
+regex = re.compile(r'\d{2}-\d{3}')
+
+for file in text_files:
+    current_file = open(file)
+    file_content = current_file.readlines()
+    for line in file_content:
+        mo = regex.search(line)
```

Comment:
Most probably you should use 'match' instead of broader 'search' but come on, don't be too pedantic ;)

'searches for any line that matches a user-supplied regular expression'